modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/block/BlockRelations.scala

```diff
@@ -0,0 +1,40 @@
+package org.tessellation.dag.l1.domain.block
+
+import cats.effect.kernel.Async
+import cats.syntax.applicative._
+import cats.syntax.eq._
+import cats.syntax.functor._
+import cats.syntax.traverse._
+
+import org.tessellation.dag.domain.block.DAGBlock
+import org.tessellation.kryo.KryoSerializer
+import org.tessellation.schema.BlockReference
+import org.tessellation.schema.transaction.TransactionReference
+import org.tessellation.security.Hashed
+import org.tessellation.security.signature.Signed
+
+object BlockRelations {
+
+  def dependsOn[F[_]: Async: KryoSerializer](
+    blockToTestAgainst: Hashed[DAGBlock]
+  )(block: Signed[DAGBlock]): F[Boolean] = dependsOn(Set(blockToTestAgainst))(block)
```

Comment:
is it needed? you put a default value (`Set.empty`) for the second argument in the second fn, so imo this one is unnecessary 🤔

Reply:
Overloaded version is used in `BlockService`.